elmslie-android/src/main/java/vivid/money/elmslie/android/manager/ElmManager.kt

```diff
@@ -0,0 +1,7 @@
+package vivid.money.elmslie.android.manager
+
+import vivid.money.elmslie.core.store.Store
+
+interface ElmManager<Event : Any, Effect : Any, State : Any> {
```

Comment:
Another store holder?)

Reply:
kind of. This abstraction is used to encapsulate all Elmslie stuff in one instance. All you need is to create this instance and Elmslie store will start and you will be able to subscribe to it.